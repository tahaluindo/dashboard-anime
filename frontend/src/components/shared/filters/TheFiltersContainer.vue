<template>
  <section>
    <slot />
  </section>
</template>

<script lang="ts">
import { defineComponent } from "vue";

export default defineComponent({});
</script>

<style lang="postcss"></style>
