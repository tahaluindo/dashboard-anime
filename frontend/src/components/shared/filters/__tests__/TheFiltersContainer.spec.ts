import { mount } from "@vue/test-utils";
import TheFiltersContainer from "../TheFiltersContainer.vue";

describe("TheFiltersContainer component test", () => {
  it("should render the default component", async () => {
    const wrapper = mount(TheFiltersContainer, {
      shallow: true,
    });

    expect(wrapper.html()).toMatchSnapshot();
  });
});
