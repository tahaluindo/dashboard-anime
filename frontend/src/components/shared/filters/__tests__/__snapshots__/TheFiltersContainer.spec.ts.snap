// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TheFiltersContainer component test should render the default component 1`] = `"<section></section>"`;
